.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - node
```

Comment:
Is it possible to define the version of Node used? I see from the tests that just ran that it's using 12.2.0 (presumably latest)

Reply:
its now using 10.15.3